pages/index.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div class="container">
+    <h1 class="title">Pokemon Familiy</h1>
+    <router-link to="/pokemons/favourites" class="favourites-link"
+      >Go to Favourites</router-link
+    >
+    <template v-if="data">
```

Comment:
do you need this `v-if` section? 

You could do: `v-for="pages in data?.pages"` and it shouldn't show anything if its undefined